CHANGELOG.md

```diff
@@ -8,7 +8,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
-...
+### Added
+
+- Added support for setting `--exclude-dir` CLI options that allow you to
```

Comment:
Style: no blank lines after H3s
```suggestion
### Added
- Added support for setting `--exclude-dir` CLI options that allow you to
```
Thanks for keeping the CHANGELOG up to date though!